admin/test/test_installer.py

```diff
@@ -0,0 +1,103 @@
+# Copyright ClusterHQ Inc.  See LICENSE file for details.
+"""
+Tests for ``admin.installer``.
```

Comment:
:grey_question: Other packages in flocker I think test internal modules in their own module (suggesting these tests should be moved to 'test/test_images.py').

I think there is some value of keeping your internal tests separate from your external tests. That way someone who uses your API can just look a test_installer.py (your external tests) to see how to use your public APIs, and your internal tests are only seen by people working within the package.

No change required, but something to consider.